### PR TITLE
feat: blog post about missing nfts

### PR DESCRIPTION
**Pull request by jchris**
blog post with NFT Port, should ship on Tuesday Apr 5th

### CODE REVIEW

**Review by dchoi27 — APPROVED** [review]
LGTM! Main thing that comes to mind that would be good to add is a reference to storage layer maximalism (https://nft.storage/blog/post/2021-12-14-storage-layer-maximalism/) - the importance of using IPFS because then you don't have to trust any one storage system (this is important to hit as defense vs. single systems that claim to be permanent but lock you in). Otherwise might expose ourselves to the "IPFS is not permanent" meme - we should just make sure folks realize that's not the point of IPFS, and in actuality IPFS helps with this problem.

Relatedly, might be good to mention niftyfixes (though I know thinking on it is still early days)

**Comment by alanshaw** [comment]
@dchoi27 there is already a link to the storage layer maximalism blog post - did you just want to add the text also?

File: packages/website/posts/2022-04-05-missing-nfts.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: 'Data integrity and $345 million worth of missing NFTs'
+description: We are starting an effort to archive NFTs as they are linked from the blockchain.
+author: Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/97632209/160728423-2011961c-faac-477b-a60c-50ae64a49767.png
```
Comment (alanshaw) [inline packages/website/posts/2022-04-05-missing-nfts.mdx:5]:
```suggestion
thumbnail: https://bafybeicqg4dyrc7m35unrlm6qp37ulbi7xwynzq67neyf2tqss37mrz4qa.ipfs.nftstorage.link/missing-nfts-hero.png
```

**Review by alanshaw — APPROVED** [review]

**Comment by dchoi27** [comment]
> @dchoi27 there is already a link to the storage layer maximalism blog post - did you just want to add the text also?

Ah missed that. Yeah what's currently there works for me (might be worth using the word IPFS to emphasize it's a feature of IPFS.

FYI - NFTPort had some other suggested changes that @jchris will probably update this PR with

**Pull request merged**